presto-base-arrow-flight/src/main/java/com/facebook/plugin/arrow/ArrowTypeHandle.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.plugin.arrow;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class ArrowTypeHandle
+{
+    private final int jdbcType;
+    private final String jdbcTypeName;
+    private final int columnSize;
+    private final int decimalDigits;
+    private final Optional<Integer> arrayDimensions;
+
+    @JsonCreator
+    public ArrowTypeHandle(
+            @JsonProperty("jdbcType") int jdbcType,
+            @JsonProperty("jdbcTypeName") String jdbcTypeName,
```

Comment:
We can't have references to JDBC in this connector.

Reply:
I have removed JDBC re references